### PR TITLE
Add more irregular words

### PR DESCRIPTION
**Pull request by hkdobrev**
Any help with the tests is appreciated!

### CODE REVIEW

File: lib/inflections/en.php [hunk 1]
```diff
@@ -39,6 +39,9 @@
 	->plural('/^(ox)$/i', '\1en')
 	->plural('/^(oxen)$/i', '\1')
 	->plural('/(quiz)$/i', '\1zes')
+	->plural('/^is$/', 'are')
```
Comment (olvlvl) [inline lib/inflections/en.php]: `is`, `was`, `has` are verbs, not nouns. Please remove them.

**Comment by olvlvl** [comment]
Thanks for your contribution! I left a comment, it should fix the issues. I'm in an hackathon until Saturday. If the problem persist I'll have a look next week.

File: lib/inflections/en.php (hunk 1 above)
Comment (hkdobrev) [reply lib/inflections/en.php]: @olvlvl I realise they are verbs, but they are quite useful when showing notifications in a website and you have a verb based on a subject which could be a number of items of 0, 1 or more than 1.

> 1 product was added to your cart

> 2 products were added to your cart

I'd use an inflector here on both `product`/`products` and `was`/`were`.

**Comment by hkdobrev** [comment]
@olvlvl I don't get why the
```
	->singular('/^are$/', 'is')
	->singular('/^were$/', 'was')
	->singular('/^have$/', 'has')
```
would break the tests as I have added `^` and `$`. Am I missing something? I suspect there could be a deeper issue behind this.

**Comment by olvlvl** [comment]
Please rebase.

**Comment by hkdobrev** [comment]
> Please rebase.
✅  Done.

File: lib/inflections/en.php [hunk 2]
```diff
@@ -66,19 +69,34 @@
...
+	->irregular('curriculum', 'curricula')
+	->irregular('foot', 'feet')
+	->irregular('genus', 'genera')
+	->irregular('goose', 'geese')
     ->irregular('leaf', 'leaves')
     ->irregular('loaf', 'loaves')
+	->irregular('man', 'men')
+	->irregular('move', 'moves')
+	->irregular('nucleus', 'nuclei')
 	->irregular('octopus', 'octopuses')
```
Comment (johnrhunt) [inline lib/inflections/en.php]: 👍

**Pull request closed without merging**